customizations.md

```diff
@@ -25,113 +23,124 @@ Examples of data include setup or initialization data, such as engine parameters
 
-As these images are produced on demand by the director repository, primaries SHOULD download all delta and / or encrypted images only from that source. After full verification of metadata, primaries SHOULD also check whether delta images match the targets metadata from the director repository, just as they check whether encrypted images match the targets metadata from the director repository when using non-delta images.
+As these images are produced on demand by the Director repository, Primaries SHOULD download all delta and / or encrypted images only from that source. After full verification of metadata, Primaries SHOULD also check whether delta images match the Targets metadata from the Director repository, just as they check whether encrypted images match the Targets metadata from the Director repository when using non-delta images.
 
 Finally, in order to install a delta image, an ECU SHOULD take one of the actions described in Table 1, depending on whether or not the delta image has been encrypted, and if the ECU has additional storage. Note that the OEM MAY use stream ciphers in order to enable on-the-fly decryption by ECUs that have no additional storage space. In this case, the ECU would decrypt the delta image as it is downloaded, then follow the remainder of the steps in the third box below.
 
-### TODO Recreate and insert Table D.2a from google doc and label it as Table 1.
+### TODO Write rationale for the SHOULDs throughout the section above.
 
-                                                                                                                                                                                                                 |
+### TODO Recreate and insert Table D.2a from google doc and label it as Table 1.
+                                                                                                                                                                                                               |
 ### Dynamic delta updates vs. precomputed delta updates
 
-There are two options when computing delta updates. Delta updates can be computed dynamically for each ECU during the installation process (dynamic delta updates), or possible delta images can be precomputed before installation begins (precomputed delta updates). The process for describing both types of updates appears below in the subsection on custom installation instructions.
+There are two options when computing delta updates. Delta updates can be computed dynamically for each ECU during the installation process (dynamic delta updates), or possible delta images can be precomputed before installation begins (precomputed delta updates). The process for describing both types of updates appears below in the subsection on Custom installation instructions.
```

Comment:
I capitalized it because its the name of a subsection on another page to which this will be links. Subheads. I'm trying to be consistent with the style used in the Standard in which subsection titles capitalized just the first word. I'm still not quite sure how to do links to other sections. I'm assuming the text will be highlighted in some manner when this is done and then it will not look so weird.

Reply:
If its just inserting a simple link, i think I can take care of that. I'm not sure about linking to a particular subsection. I think I would rather get everything else committed and then do the links by themselves.

Reply:
As you like. But if you approve my suggested change here, then the work in this case is already done. :)